tests/behat/features/file.feature

```diff
@@ -0,0 +1,22 @@
+@d7
+Feature: Check that FileTrait works
+
+  @api
+  Scenario: Assert "Given managed file:"
+    When I am logged in as a user with the "administrator" role
+    Given managed file:
+      | path                                                        |
+      | example_document.pdf                                        |
+      | example_image.png                                           |
+      | example_audio.mp3                                           |
+      | https://www.sample-videos.com/img/Sample-jpg-image-50kb.jpg |
+    Then "example_image.png" file object exists
```

Comment:
please add `Then :file_name file object exists` assertion for each file from the list